codes.md

```diff
@@ -220,6 +229,8 @@ Kate
        up with new grouping for writing implements
      - [Session 031](https://osf.io/cy3s4/) - item details considered during
        drawing to clarify depiction
+     - [Session 033](https://osf.io/uj34p/) - icon detail/symbole choices not
+       thought about until drawing
```

Comment:
Oo there was a lot of this "didn't think about the shapes/icons til I started drawing." That feeling of _I'm stuck, I didn't think of this_ is close to what I mean in the code "People run into constraints with their existing sketch." I'd like to review the interviews and see who verbally questions their drawing choices (either in-process or after)

Reply:
I agree diving into this would also help us understand mental models and evolution better. I assume this is for a later PR though.